ci/test/04_install.sh

```diff
@@ -92,6 +92,15 @@ export DIR_FUZZ_IN=${DIR_QA_ASSETS}/fuzz_seed_corpus/
 
 DOCKER_EXEC mkdir -p "${BASE_SCRATCH_DIR}/sanitizer-output/"
 
+if [[ ${USE_MEMORY_SANITIZER} == "true" ]]; then
+  DOCKER_EXEC "update-alternatives --install /usr/bin/clang++ clang++ \$(which clang++-8) 100"
+  DOCKER_EXEC "update-alternatives --install /usr/bin/clang clang \$(which clang-8) 100"
+  DOCKER_EXEC "mkdir -p ${BASE_SCRATCH_DIR}/msan/build/"
+  DOCKER_EXEC "git clone --depth=1 https://github.com/llvm/llvm-project ${BASE_SCRATCH_DIR}/msan/llvm-project"
```

Comment:
Not sure if we want to compile a random commit from the default branch. Is msan required to be built from source? Would it help if you used the Ubuntu focal libcxx like in #19041?

Reply:
Pinning against a specific commit in the LLVM repo sounds like a good idea :)

Unfortunately I am not aware of any way to to achieve full MemorySanitizer instrumentation without linking against an MSan-instrumented C++ standard library, and I'm now aware of any way of doing so without compiling from source.

From various parts of the MemorySanitizer documentation: _"Full MemorySanitizer instrumentation is very difficult to achieve." […] "If you want MemorySanitizer to work properly and not produce any false positives, you must ensure that all the code in your program and in libraries it uses is instrumented (i.e. built with -fsanitize=memory). In particular, you would need to link against MSan-instrumented C++ standard library. We recommend to use libc++ for that purpose."_

Reply:
Now pinning against `e5b8772756737e41cb1e8ee1a5a33cb3d8a25be6` (https://github.com/llvm/llvm-project/commit/e5b8772756737e41cb1e8ee1a5a33cb3d8a25be6) which happens to be the current `master` at the time of writing.

Reply:
Oh, so the Ubuntu focal libcxx does not ship with  msan?

Reply:
Nope, I don't think so (in other words: it is not built with MSan-instrumentation AFAIK).

Reply:
Now pinning to v10.0.0 instead which feels less random :)